effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -125,13 +139,23 @@ object JSRunner extends Runner[String] {
    */
   def build(path: String)(using C: Context): String =
     val out = C.config.outputPath().getAbsolutePath
-    val jsFilePath = (out / path).unixPath
+    val jsFilePath = (out / path).canonicalPath.escape
     // create "executable" using shebang besides the .js file
-    val jsScriptFilePath = jsFilePath.stripSuffix(s".$extension")
     val jsScript = s"require('${jsFilePath}').main()"
-    val shebang = "#!/usr/bin/env node"
-    IO.createFile(jsScriptFilePath, s"$shebang\n$jsScript", true)
-    jsScriptFilePath
+    os match {
+      case OS.POSIX =>
+        val shebang = "#!/usr/bin/env node"
+        val jsScriptFilePath = jsFilePath.stripSuffix(s".$extension")
+        IO.createFile(jsScriptFilePath, s"$shebang\n$jsScript", true)
+        jsScriptFilePath
+
+      case OS.Windows =>
+        val jsMainFilePath = jsFilePath.stripSuffix(s".$extension") + "__main.js"
+        val exePath = jsFilePath.stripSuffix(s".$extension")
+        val batScript = s"node $jsMainFilePath"
```

Comment:
This variable is unused, right?

Reply:
Ah, yes, this is left-over from an older version.
```suggestion
```